IlmBase/config/LibraryDefine.cmake

```diff
@@ -94,6 +94,7 @@ function(ILMBASE_DEFINE_LIBRARY libname)
     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
     LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    INCLUDES DESTINATION include
```

Comment:
should this and the others below not be ${CMAKE_INSTALL_INCLUDEDIR} to match the other gnuinstalldirs we use? Yes, this defaults to include, but just to be pedantic, it should probably match the use of the gnuinstalldirs variables...

Reply:
That makes sense, rebased and changed